### ISSUE TITLE
Print Composer integration

### ISSUE THREAD

**Issue by carolinux** [anitagraser/TimeManager#138]

**Comment by anitagraser** [comment]
I think the main advantage of being able to export print compositions rather than just the main map window would be that one could add a legend and more labels. That said, I would not start implementing this now because the print composer code will be changed - probably including an API break - in the next months (I think Nyall is working on it).

**Comment by carolinux** [comment]
Yes, that's my motivation :-) Thanks for the heads up, I suppose I will speak to him in the upcoming developer meeting then.

**Comment by nirvn** [comment]
I'd add that the other main advantage of going through a composer is that you have pixel-level control of the width and height of your export (vs achieving an approximate radio by resizing panels to affect the size of the main canvas)

**Comment by anitagraser** [comment]
FYI: The Crayfish plugin already has Print Composer support, for a demo see https://www.youtube.com/watch?v=bMJlqFJznjQ&feature=youtu.be

**Comment by nyalldawson** [comment]
For reference: QGIS 3.14 brings the ability to set a fixed time range for a layout map item. It doesn't handle animation type exports of layouts, but it does allow things like time-based map atlases or "small multiples" showing different temporal ranges.
